runtime/onert/core/include/ir/operation/Loss.h

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I told @ragmani, why `ir/operation/Loss.h` needs to be introduced. I agreed to introduce this file:

- there are some required methods from `ir/operation`
- to be consistent